content/20.pharmaceuticals.md

```diff
@@ -790,25 +790,45 @@ Dendritic cells are potent antigen-presenting immune cells that easily take up m
+Among the 95 cases reported in the first interim analysis, 16% occurred in older adults [@url:https://investors.modernatx.com/news-releases/news-release-details/modernas-covid-19-vaccine-candidate-meets-its-primary-efficacy].
+Additionally, 9.7% of participants identified a Black or African-American background as well as 20% who identified Hispanic or Latino ethnicity [@url:https://www.fda.gov/media/144434/download].
+Among the 95 cases, 12.6% occurred in participants identifying a Hispanic or Latino background and 4% in participants reporting a Black or African-American background [@url:https://investors.modernatx.com/news-releases/news-release-details/modernas-covid-19-vaccine-candidate-meets-its-primary-efficacy].
+While the sample size of 95 cases is small relative to the study population of 30,000, these results suggest that the vaccine is likely to effective in people from a variety of backgrounds.
+All the same, the publicly available numbers suggest that this vaccine is likely to be highly useful in containing SARS-Cov-2.
+
+In-depth safety data was released by ModernaTX as part of their application for an EUA from the FDA.
+A large percentage of participants reported adverse effects when solicited, and these reports were higher in the vaccine group than in the placebo group (94.5% versus 59.5%, respectively) [@url:https://www.fda.gov/media/144434/download].
+Some of these events met the criteria for grade 3 (local or systemic) or grade 4 (systemic only) toxicity [@url:https://www.fda.gov/media/144434/download].
+The most common local adverse reaction was pain at the injection site, reported by 83.7% of participants receiving the first dose of the vaccine and 88.4% upon receiving the second dose, compared to 19.8% and 19.8% and 17.0%, respectively, of patients in the placebo condition [@url:https://www.fda.gov/media/144434/download].
+Fewer than 5% of vaccine recipients reported Grade 3 pain at either dosage.
+The other most frequently observed local reactions included erythema, swelling, and lymphadenopathy [@url:https://www.fda.gov/media/144434/download].
+For systemic adverse reactions, fatigue was the most common [@url:https://www.fda.gov/media/144434/download].
+Among participants receiving either dose of the vaccine, 68.5% reported fatigue compared to 36.1% participants receiving the placebo [@url:https://www.fda.gov/media/144434/download].
+The level of fatigue experienced was usually fairly mild, with only 9.6% and 1.3% of participants in the vaccine and placebo conditions, respectively, reporting grade 3 fatigue [@url:https://www.fda.gov/media/144434/download], which corresponds to significant interference with daily activity [@url:https://www.fda.gov/media/73679/download].
+While no publication has yet been released and the company's petition for an EUA is still outstanding, these data overall provide support for the early promising results released by ModernaTX.
```

Comment:
We have the paper now (DOI: 10.1056/NEJMoa2035389) and the EUA (https://www.fda.gov/emergency-preparedness-and-response/coronavirus-disease-2019-covid-19/moderna-covid-19-vaccine)

You need help updating?

Reply:
I just pushed some updates! I don't totally understand the EMA from the EU so I would especially appreciate it if you don't mind making sure that language is correct! "marketing authorisation" is a little different than the other terminology!

Reply:
No worries, I will look